app/discover/procurement/page.tsx

```diff
@@ -0,0 +1,400 @@
+</Text>
+
+<Text>
+If you want to enable or disable the functionalities available to the end-user of your Oskari-based application, changes to the code are relatively quick.
+</Text>
+
+<h3>Developing existing functionalities</h3>
+
+<Text>
+In some cases you might want to develop functionalities that already exist either in the Oskari sample application or that are made by other Oskari users. In this case you should first <Link href='/contribute'>contact the project coordinators of Oskari</Link> to find out whether or not the functionality is still actively maintained.
+</Text>
+
+<Text>
+In other cases a certain functionality is popular amongst other Oskari users and other organisations might have similar wishes to develop it further, but that requires communication between organisations.
+</Text>
+
+<Text>
+On the other hand, the functionality might not be used by anyone and hasn&apos;t been maintained, so the project steering committee of Oskari is about to drop it out of the codebase. 
+</Text>
+
+<h3>Creating new functionalities</h3>
+
+<Text>
+Before creating new functionalities, see the list of existing functionalities in <Link href='https://oskari.org/documentation/api/bundles/latest'>the bundle documentation</Link> and <a href='https://github.com/oskariorg/oskari-frontend-contrib'>the functionalities and customisations done by other Oskari users (the unofficial Oskari bundles created by the Oskari community).</a> One should also think about joint development: could this be something that some other organisation using Oskari could also wish to have?
+</Text>
+
+<Text>
+If you want to create new functionalities for Oskari, it might be a good idea to gather together all the ideas that you have and hand them over to the development team. Thus they can see which are top of your priorities now and what kind of things to expect next - and also suggest to you which of your ideas could be fulfilled at the same time quite easily. By coming up with multiple ideas at once they can also take the upcoming ideas into consideration when developing the first new functionalities.
+</Text>
+
+<Text>
+With new functionalities, one should remember that they must be maintained. When you create new functionalities to your Oskari service, it could be beneficial <Link href='/contribute'>to contact the project coordinators</Link> beforehand. They might be able to offer feedback and help coordinate development of similar functionalities if you contact them before you start the development.Your developers can get best practices for the development process of Oskari and further ideas on how to create the functionalities. 
```

Comment:
```suggestion
With new functionalities, one should remember that they must be maintained. When you create new functionalities to your Oskari service, it could be beneficial <Link href='/contribute'>to contact the project coordinators</Link> beforehand. They might be able to offer feedback and help coordinate development of similar functionalities if you contact them before you start the development. Your developers can get best practices for the development process of Oskari and further ideas on how to create the functionalities. 
```